doc/source/whatsnew/v0.14.0.rst

```diff
@@ -496,86 +502,87 @@ See also issues (:issue:`6134`, :issue:`4036`, :issue:`3057`, :issue:`2598`, :is
 
    You should do this:
 
-   .. code-block:: python
+  .. code-block:: python
 
-      df.loc[(slice('A1','A3'),.....),:]
+    df.loc[(slice('A1', 'A3'), .....), :]
```

Comment:
this is invalid python code obviously, the reason is not failing is because it's not indented and it's rendered not as code but as a text paragraph. Inside `.. code-block:: python` code statements need to start with `>>>`. In this case you'll have to use a `noqa: E901` to make the validation run. Also, I'd leave just `...` for the `.....`.